metricshub-doc/src/site/markdown/connector-design-guide/connector-object-references.md

```diff
@@ -0,0 +1,173 @@
+keywords: guide, connector, connector, object, reference
+description: This page shows how to reference connector objects such as sources, entries, columns and files, etc. inside the YAML document.
+
+# Connector Object References
+
+This page shows how to reference connector objects such as sources, entries, columns and files, etc. inside the YAML document.
+
+* [Source Reference](#Source_Reference)
+    * [Format](#source-format)
+    * [Example](#source-example)
+* [Entry Reference](#Entry_Reference)
+    * [Format](#entry-format)
+    * [Example](#entry-example)
+* [Column Reference](#Column_Reference)
+    * [Format](#column-format)
+    * [Example](#column-example)
+* [File Reference](#File_Reference)
+    * [Format](#file-format)
+    * [Example](#file-example)
+* [Mono-Instance Reference](#Mono-Instance_Reference)
+    * [Format](#mono-instance-format)
+    * [Example](#mono-instance-example)
+* [Translation Table Reference](#Translation_Table_Reference)
+    * [Format](#translation-table-format)
+    * [Example](#translation-table-example)
+* [Awk Script Reference](#Awk_Script_Reference)
+    * [Format](#awk-script-format)
+    * [Example](#awk-script-example)
+
+## Source Reference
+
+### <a id="source-format" />Format
+```yaml
+${source::<fullSourcePath>}

```

Comment:
Do we always use `fullSourcePath`? Aren't they other scenarios where we use a relative path? If it is the case, can you please detail a little bit more using examples.

Reply:
The source path is not a File path, but the path of the source within the connector, eg: `${source::monitors.disk_controller.discovery.sources.source_discovery}`.
There is example just in the section source-example, just below this section.